README.md

```diff
@@ -22,49 +22,133 @@ cargo build --release
 ```
 
 ### Run Manually
- 
-Example command option for launch one of node in 3 of 5 Signer network.
 
-Must launch `tapyrus-core` and `redis` ahead.
+Here, we build a Tapyrus Signer Network that can generate a block if 2 of 3 signers agree. 
+(Current implementation has an [issue](https://github.com/chaintope/tapyrus-signer/issues/29) for that threshold process.)
+For build this network, there are following steps. 
+
+#### 1. Generate singing keys.
+
+There are 3 signers named Alice, Bob, Carol. Each signer needs own private key and public key on secp256k1 elliptic curve.
+
+Here we use below key pairs. Don't use these keys for serious use.
+
+```
+Key pair for Alice:
+private key: 53f0d2efbc1e6aae7bcfc762a20a3df0b700cf9808dbdc0855dca6481486ed61
+WIF: cQPsVjpiP35ceNqLWDohBdnxedsMzCoX3NFaiSjYvSyFysNEKyBt
+public key: 0341c6dc48817c840e17428c50cc9fe71802a2d3a2a36519f63dabc10b5713acf2
+
+Key pair for Bob:
+private key: ad11db5745d909cf5bb769fe45acdd435884637fb1fca2cfb6e212deff62fe17
+WIF: cTP8JDfEp6s7UYnyexESXSMYi3XSmxRCxS4wFbYwU5eMgvC1kHXk
+public key: 03cf6ababa85c0687f1d04bb3446ad24da879519f72ef037e900667cc3cdf1e904
+
+Key pair for Carol:
+private key: 74431a3cdb2cc82dfc32a691d9f7e68da7675ba81e6833e00d744169d74faae6 
+WIF: cRUhZ5WxmiJ5f87gFoztWRztUwSk7cTfRick4z2Qg7zJnZjdnuJj
+public key: 02d7facf8f7b3182dc03d5888fdf78cc5c2d0a5ce14559ffbaa3bab9f86272c591
+```
+
+#### 2. Create aggregated key pair.
+
+In general, private Key is confidential. Essentially, such a data as a private key should not be exist in real Tapyrus 
+Blockchain Network. This key is created only for test temporally. In a future version, it is going not to need to 
+generate aggregated private key.
+
```

Comment:
What exactly is a temporary test?  Unnecessary private key aggregation should be removed to avoid confusion.

Reply:
Sorry, it was not kindness. 

How about this?

> Essentially, an aggregated private key must not be exist in real Tapyrus Blockchain Network. This document's target is for test on your local environments. So, in order to make it easy to setup, we prepare aggregated private key for to use in step "4. Generate genesis block for this network".
> 
> In real network, all proofs for the blocks are created by secret sharing scheme, so there is not need to get aggregated private key. Regarding public key, it is needed to verify proofs, and it is created in distributed environment by secret sharing. If you want to know more details of secret sharing scheme which is used in Tapyrus Signer Network, you can get paper [here](cacr.uwaterloo.ca/techreports/2001/corr2001-13.ps).